### PR TITLE
patient home page updated

### PR DESCRIPTION
**Pull request by ImArjunAsok**

### CODE REVIEW

File: Client/src/app/footer/footer.component.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
```
Comment (mishelshaji) [inline Client/src/app/footer/footer.component.html:1]: Remove CDN links from HTML page.

File: Client/src/app/patient/home/home.component.css [hunk 2]
```diff
@@ -58,12 +56,52 @@
...
-    }
-    
-    .searcharea {
-        width: 100%;
-    }
+.result-dept {
+    background: linear-gradient(108.23deg, rgba(180, 229, 218) 6.41%, rgba(99, 218, 189, 0.3) 22.08%, rgba(255, 255, 255, 0.3) 40.5%, rgba(246, 253, 248, 0.299921) 61.69%, rgba(224, 247, 229, 0.299713) 81.03%, rgba(86, 210, 113, 0.298438) 94.85%), url("../../../assets/images/resultpage.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+
```
Comment (mishelshaji) [inline Client/src/app/patient/home/home.component.css:63]: Remove blank line.

File: Client/src/app/patient/home/home.component.html [hunk 3]
```diff
@@ -9,11 +9,23 @@ <h2>Search Doctor, Make an Appointment</h2>
             <div class="d-flex">
```
Comment (mishelshaji) [inline Client/src/app/patient/home/home.component.html:9]: Where is the form tag?

File: Client/src/app/patient/home/home.component.ts [hunk 4]
```diff
@@ -6,5 +6,42 @@ import { Component } from '@angular/core';
   styleUrls: ['./home.component.css']
 })
 export class HomeComponent {
-
+    doctors = [
```
Comment (mishelshaji) [inline Client/src/app/patient/home/home.component.ts:9]: Why is it defined here?

**Review by mishelshaji — CHANGES_REQUESTED** [review]

File: Client/src/app/patient/home/home.component.ts (hunk 4 above)
Comment (ImArjunAsok) [reply Client/src/app/patient/home/home.component.ts:9]: its okay

**Review by mishelshaji — APPROVED** [review]

**Pull request merged**